test/e2e/cluster_upgrade_runtimesdk.go

```diff
@@ -478,9 +490,33 @@ func runtimeHookTestHandler(ctx context.Context, c client.Client, namespace, clu
 		fmt.Sprintf("ClusterTopology reconcile did not unblock after updating hook response: hook %s still blocking", hookName))
```

Comment:
I think we have to be careful with the error message given that we will hit this case if either:
* the hook is still blocking or
* (create failed | upgrade failed | delete failed)

This might become confusing for folks trying to debug a failed test run